app.js

```diff
@@ -0,0 +1,18 @@
+const express = require("express");
+const { getCategories } = require("./controller/controller");
+const {
+  handleInternalErrors,
+  handleCustomErrors,
+} = require("./controller/errors.controllers");
+
+const app = express();
+
+app.use(express.json());
+
+app.get("/api/categories", getCategories);
```

Comment:
probably need app.all to handle the invalid endpoint, so we can send an error message